.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove your .vscode files from the repo. Potentially add .vscode to .gitignore

Reply:
Sounds good. I will remove this.